src/Microsoft.AspNet.Mvc.Razor/Resources.resx

```diff
@@ -141,6 +141,12 @@
   <data name="MvcRazorCodeParser_OnlyOneModelStatementIsAllowed" xml:space="preserve">
     <value>Only one '{0}' statement is allowed in a file.</value>
   </data>
+  <data name="RazorPage_ThereIsNoActiveWritingScopeToEnd" xml:space="preserve">
+    <value>There is no active writing scope to end.</value>
+  </data>
+  <data name="RazorPage_YouCannotFlushWhileInAWritingScope" xml:space="preserve">
+    <value>You cannot flush while inside a writing scope.</value>
```

Comment:
@pranavkm Correct, I came here now because I have no idea why this error is coming and this was the only hit on Google. Better wording would be good.